src/commonmark/en/content/user/data-visualizer.md

```diff
@@ -2,54 +2,323 @@
 
 <!--DHIS2-SECTION-ID:data_visualizer-->
 
+![](resources/images/data-visualizer/data-visualizer-overview.png)
+
 ## Creating and editing visualizations
 
 When you open the data-visualizer app from the dhis2 menu, you are presented with a blank slate and you can start creating your visualization right away.
 
 ![](resources/images/data-visualizer/data-visualizer-new.png)
 
+### Select visualization type
+
 Select the desired visualization type from the selector in the upper left corner:
 
 ![](resources/images/data-visualizer/data-visualizer-visualization-type.png)
 
-From the dimension menu on the left you can select the dimension you want
-to show in your visualization, including data, period, organisation units and
-dynamic dimensions.
+<br/>
+<table>
+<colgroup>
+<col style="width: 33%" />
+<col style="width: 66%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th><p>Visualization type</p></th>
+<th><p>Description</p></th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td><p>Column</p></td>
+<td><p>Displays information as vertical rectangular columns with lengths proportional to the values they represent.</p>
+<p>Useful when you want to, for example, compare performance of different districts.</p></td>
+</tr>
+<tr class="even">
+<td><p>Stacked column</p></td>
+<td><p>Displays information as vertical rectangular columns, where bars representing multiple categories are stacked on top of each other.</p>
+<p>Useful when you want to, for example, display trends or sums of related data elements.</p></td>
+</tr>
+<tr class="odd">
+<td><p>Bar</p></td>
+<td><p>Same as column, only with horizontal bars.</p></td>
+</tr>
+<tr class="even">
+<td><p>Stacked bar</p></td>
+<td><p>Same as stacked column, only with horizontal bars.</p></td>
+</tr>
+<tr class="odd">
+<td><p>Line</p></td>
+<td><p>Displays information as a series of points connected by straight lines. Also referred to as time series.</p>
+<p>Useful when you want to, for example, visualize trends in indicator data over multiple time periods.</p></td>
+</tr>
+<tr class="even">
+<td><p>Area</p></td>
+<td><p>Is based on a line (above), with the space between the axis and the line filled with colors and the lines stacked on top of each other.</p>
+<p>Useful when you want to compare the trends of related indicators.</p></td>
+</tr>
+<tr class="odd">
+<td><p>Pie</p></td>
+<td><p>Circle divided into sectors (or slices).</p>
+<p>Useful when you want to, for example, visualize the proportion of data for individual data elements compared to the total sum of all data elements.</p></td>
+</tr>
+<tr class="even">
+<td><p>Radar</p></td>
+<td><p>Displays data on axes starting from the same point. Also known as spider chart.</p></td>
+</tr>
+<tr class="odd">
+<td><p>Gauge</p></td>
+<td><p>Semi-circle which displays values out of 100 %.</p></td>
+</tr>
+<tr class="even">
+<td><p>Year over year (line)</p></td>
+<td><p>TODO: NEEDS INFO </p></td>
```

Comment:
This + the next 3 needs a short one-liner to give a brief explanation of what it is and what it's good for. Please see the other types above for an example of the format. @janhenrikoverland would you mind helping me with these?